plugins/generator-1.18.2/forge-1.18.2/generator.yaml

```diff
@@ -56,7 +56,7 @@ base_templates:
     name: "@RESROOT/META-INF/accesstransformer.cfg"
     condition_any:
       - ${w.hasGameRulesOfType('Number') || w.hasGameRulesOfType('Logic')}
-      - ${w.hasElementsOfType('dimension') || w.hasElementsOfType('biome')}
+      - ${w.hasElementsOfType('dimension') || w.hasElementsOfType('biome')} || ${w.hasElementsOfType('villagerprofession')}

```

Comment:
How is this supposed to work? Move it to a separate line (after fishing rods).

Reply:
Done!